objects/src/accounts/delta/vault.rs

```diff
@@ -233,6 +233,11 @@ impl FungibleAssetDelta {
         self.add_delta(asset.faucet_id(), -amount)
     }
 
+    /// Returns the number of fungible assets affected in the delta.
+    pub fn count(&self) -> usize {
```

Comment:
nit: we usually use `num_` prefix rather than `count`. So, here it could be `num_assets`.